modules/ravestate_nlp/triple.py

```diff
@@ -71,7 +71,7 @@ def match_either_lemma(self, pred: Optional[Set[str]] = None, subj: Optional[Set
             return True
         if subj and self._subject.lemma_ in subj:
             return True
-        if obj and self._object.lemma_ in obj:
+        if obj and self._object.lemma_ in obj or self._object.lemma_ in [str(child) for child in self._object.children]:
```

Comment:
Note:
- It should also work for subject
- It will crash when object is a `QuestionWord` instead of a `spacy.Token`
- Use child.lemma_ instead of str(child)
- Please add a test 😊

Reply:
Use `any(child.lemma_ in obj for child in self._object.children)`.

What is the purpose of checking whether the object is contained in the children??

Reply:
Also, are you sure that `.children` contains ALL subtokens?

Reply:
Added the changes and tests